doc/rados/operations/placement-groups.rst

```diff
@@ -153,7 +153,9 @@ The output will resemble the following::
 - **NEW PG_NUM** (if present) is the value that the system recommends that the
   ``pg_num`` of the pool should be. It is always a power of two, and it
   is present only if the recommended value varies from the current value by
-  more than the default factor of ``3``.
+  more than the dynamic scaling threshold. The dynamic scaling threshold defaults
+  to the factor of ``3``, but it is automatically set to 1.0 when the recommended
+  NEW PG_NUM is 512 or 1024, and to 2.0 when it is 2048.
```

Comment:
Should this be "2048 or larger"?

Note to self and others: this should be backported only in conjunction with the below code.

Reply:
If new pg_num is more than 2048 then default scaling threshold will apply.